src/app/modules/practitioner-registration/pages/facility-info/facility-info.component.ts

```diff
@@ -99,11 +100,16 @@ export class FacilityInfoComponent extends BcpBaseForm implements OnInit, AfterV
 
         }, error => {
           console.log('ARC apiService onerror', error);
-          this.containerService.setIsLoading(false);
+          this.handleError();
         });
     }
   }
 
+  private handleError(): void {
+    this.systemDownError = true;
+    this.containerService.setIsLoading(false);
+  }
+
   private handleValidation(isValid: boolean): void {
     this.showValidationError = !isValid;
```

Comment:
Should probably add this.systemDownError = false;  

Just in case they try again and it goes through.